packages/js/internal-js-tests/src/setup-react-testing-library.js

```diff
@@ -3,3 +3,21 @@
  */
 import '@wordpress/jest-console';
 import '@testing-library/jest-dom';
+
+/**
+ * Ignore messages that match the pattern `Store ".*" is already registered.`
+ * It's expected to have duplicate registration in the test environment as there are multiple instances of the same store such as core-data, core/block-editor, etc.
+ */
+const consoleErrorSpy = jest
```

Comment:
Do you think this will eventually be resolved, or would this potentially always be required?